hazelcast-enterprise/Dockerfile

```diff
@@ -35,32 +39,44 @@ ENV HZ_HOME="${HZ_HOME}" \
     CLASSPATH="" \
     JAVA_OPTS=""
 
+LABEL name="Hazelcast IMDG Enterprise" \
+      maintainer="info@hazelcast.com" \
+      vendor="Hazelcast, Inc." \
+      version="8.1" \
```

Comment:
What is this `version`? I know we had it in `hazelcast-openshift`, but is it still mandatory to have?

Reply:
Actually, we used it wrong all the time, it should be image/hazelcast version. And yes it is mandatory, there are tests for it and other labels at `Certification Build`. Here is the related section from [official doc](https://connect.redhat.com/zones/containers/container-certification-policy-guide#version_label_exists):
<img width="721" alt="Screen Shot 2020-12-16 at 17 28 37" src="https://user-images.githubusercontent.com/6005622/102362432-6fb25f00-3fc5-11eb-92dd-91beb3c444eb.png">